### PR TITLE
Bookstore: Add styles

### PR DESCRIPTION
**Pull request by jahongiry**
**Added styles to the project**
- Used CSS classes
- Used Zeplin design

### CODE REVIEW

File: src/components/Navigation.module.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  display: flex;
+  gap: 4rem;
+  list-style: none;
+}
+
+.lists {
+  color: rgb(152, 114, 113);
+  text-decoration: none;
+  font-size: 1.2rem;
+  font-weight: 600;
```
Comment (rotimiazeez) [inline src/components/Navigation.module.css]: - Kindly make sure your navigation menu is well aligned as seen on the Zeplin page for better UI(user interface)
- Your Design | Zeplin Design

Screenshot from 2022-07-16 07-48-16
Screenshot from 2022-07-16 07-48-35

File: src/components/book-item.js [hunk 2]
```diff
@@ -13,14 +14,14 @@ const Book = ({
...
+            <h4 className={classes.fiction}>{category}</h4>
+            <h2 className={classes.title}>{title}</h2>
             <h6>{author}</h6>
-            <div>
-              <button type="button">Comments</button>
+            <div className={classes.cre}>
+              <button type="button">Comments |</button>
               <div />
               <button
                 className="button-outline"
```
Comment (rotimiazeez) [inline src/components/book-item.js]: - Please make sure the Form and books font, colour, font size and design are the same as that of the Zeplin design
Zeplin Design | Your Design

Screenshot from 2022-07-16 09-39-03
Screenshot from 2022-07-16 08-18-04

**Review by rotimiazeez — APPROVED** [review]
# Hi @jahongiry  👏🏽, 
## Good job so far! 👍🏽 
### Summary ♨️ 
- No Linter error ✔️ 
- Good and descriptive PR 💯 
- Good README 👍🏽
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimemalaika — APPROVED** [review]
Hi @jahongiry, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**